### ISSUE TITLE
Use newer kde frameworks snap in snap package

### ISSUE THREAD

**Issue by jriddell** [flameshot-org/flameshot#1874]
The snap package uses the older core18 and kde-frameworks-5-core18 content snaps.  This has a problem e.g. https://forum.snapcraft.io/t/possible-compatibility-issue-with-kde-frameworks-5-core18-and-centos-7/26433

I've a newer one kde-frameworks-5-qt-5-15-3-core20 that uses core20. If you were interested in using this I could request it be made useable by 3rd party snaps.  Is that interesting?

**Comment by mmahmoudian** [comment]
@jriddell As far as I can recall, @borgmanJeremy has mentioned in a comment (I should search and find it) that he is in fact planning to move to the new KDE core for the Snap.

**Comment by jriddell** [comment]
ok request sent https://forum.snapcraft.io/t/allow-global-auto-connect-for-kde-frameworks-5-qt-5-15-3-core20/26470

**Comment by borgmanJeremy** [comment]
Yes moving to core20 should resolve many of our problems, but when I last checked there wasnt a release of kde-frameworks with core20.

**Comment by jriddell** [comment]
It's here and hopefully it'll get global auto connect soon so 3rd party apps can use it
https://snapcraft.io/kde-frameworks-5-qt-5-15-3-core20

**Comment by jriddell** [comment]
Oh and the kde-neon extension uses it https://snapcraft.io/docs/kde-neon-extension

**Comment by borgmanJeremy** [comment]
I did some preliminary investigation and i think core 20 and kde framework is ready to use. If someone wants to help you can use this as a reference I think: https://github.com/KDE/snapcraft-kde-applications/blob/Neon/release/spectacle/snapcraft.yaml
